com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md

```diff
@@ -272,18 +302,52 @@ HDRP includes its own purpose-built implementation for post-processing to produc
 
 ![Posterize](Images/HDRPFeatures-CustomPostProcess.png)
 
-HDRP allows you to add your own custom post processes integrated with the volume framework. They can be injected after opaque and sky object, before builtin post processes or after builtin post processes. For more information, see the [Custom Post-processing documentation](Custom-Post-Process.md).
+HDRP allows you to add your own custom post processes integrated with the volume framework. They can be injected after opaque and sky object, before the [temporal anti-aliasing](Anti-Aliasing.md#TAA) pass, before builtin post processes or after builtin post processes. For more information, see the [Custom Post-processing documentation](Custom-Post-Process.md).
+
+### Path tracing
+
+#### Path-traced depth of field
+
+![](Images/Path-traced-DOF-Feature.png)
+
+HDRP includes a depth of field mode for producing path-traced images with high-quality defocus blur. Compared to post-processed depth of field, this mode works with multiple layers of transparency and does not produce any artifacts, apart from noise typical in path traced images. You can fix this by increasing the sample count and/or using an external denoising tool.
+
+For more information about this feature, see [Depth-of-field](Post-Processing-Depth-of-Field.md).
+
+#### Accumulation motion blur and path tracer convergence APIs
+
+![](Images/Path_tracing_recording-Feature.png)
+
+HDRP includes a recording API which you can use to render effects such as high-quality accumulation motion blur and converged path-traced images. These techniques create the final "converged" frame by combining information from multiple intermediate sub-frames. The new API allows your scripts to extract the properly converged final frames and perform further processing or save them to disk.
```

Comment:
We need to separte Accumulation motion blur  and path tracer convergence APIs.

Let's move Accumulation motion blur APIs. in Camerea section just after postprocess and 

let ath tracer convergence APIs here

for accumulation you have:
HDRP includes a recording API which you can use to render effects such as high-quality accumulation motion blur. The technique create the final frame by combining information from multiple intermediate sub-frames. The new API allows your scripts to extract the properly final frames and perform further processing or save them to disk.

for pathtracer you have:

HDRP includes a recording API which you can use to render converged path-traced images. The techniques create the final "converged" frame by combining information from multiple intermediate sub-frames. The new API allows your scripts to extract the properly converged final frames and perform further processing or save them to disk.